server/__tests__/deepseek.test.ts

```
@@ -22,3 +36,29 @@ test('deepseek wrapper throws after retries', async () => {
   expect(spy).toHaveBeenCalled();
   spy.mockRestore();
 });
+
+test('deepseek throws when API key missing', async () => {
+  const original = process.env.DEEPSEEK_API_KEY;
+  delete process.env.DEEPSEEK_API_KEY;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)